src/hooks/useValidation/useValidation.js

```diff
@@ -24,10 +50,68 @@ const useValidation = (context) => {
       return joinErrors(joinedErrors, ruleErrors);
     }, {});
 
-    return errors;
+    return formatFEValidation(errors);
   }, [context, quickMarcContext]);
 
-  return { validate };
+  const formatBEValidationResponse = (response, marcRecords) => {
+    if (!response.issues) {
+      return {};
+    }
+
+    return response.issues.reduce((acc, cur) => {
+      const match = cur.tag.match(/(.{0,3})\[(\d)\]/);
+      const fieldTag = match[1];
+      const fieldIndex = parseInt(match[2], 10);
+
+      const field = marcRecords.filter(_field => _field.tag === fieldTag)[fieldIndex];
+
+      const existingIssues = acc[field?.id || MISSING_FIELD_ID] || [];
+
+      return {
+        ...acc,
+        [field?.id || MISSING_FIELD_ID]: [...existingIssues, cur],
+      };
+    }, {});
+  };
+
+  const runBackEndValidation = useCallback(async (marcRecords) => {
+    const body = {
+      fields: marcRecords.filter(record => !isLeaderRow(record)),
+      leader: marcRecords.find(isLeaderRow)?.content,
+      marcFormat: context.marcType.toUpperCase(),
+    };
+
+    const response = await validateFetch({ body });
+
+    return formatBEValidationResponse(response, marcRecords);
+  }, [context, validateFetch]);
+
+  const validate = useCallback(async (marcRecords) => {
+    let errors = {};
+
+    if (BE_VALIDATION_MARC_TYPES.includes(context.marcType)) {
+      errors = await runBackEndValidation(marcRecords);
+    } else {
+      errors = runFrontEndValidation(marcRecords);
+    }
+
+    quickMarcContext.setValidationErrors(errors);
+
+    return Promise.resolve(errors);
+  }, [quickMarcContext, context, runFrontEndValidation, runBackEndValidation]);
+
+  const hasIssuesBySeverity = (severity) => {
+    return Object.values(quickMarcContext.validationErrorsRef.current)
+      .reduce((hasError, errorsList) => {
+        return hasError || Boolean(errorsList.find(err => err.severity === severity));
+      }, false);

```

Comment:
I think this can be simplified to 
```
Object.values(quickMarcContext.validationErrorsRef.current)
    .flat()
    .some(fieldError => fieldError.severity === severity)
```